src/components/Votes.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { getUpdateVotes } from "../axios/apiQueries";
+
+const Votes = ({ review_id, votes }) => {
+  const [userVote, setUserVote] = useState(0);
+  const [updatedVotes, setUpdatedVotes] = useState(votes);
+
+  const onVoteUp = () => {
+    setUserVote(1);
+    setUpdatedVotes(updatedVotes);
+    getUpdateVotes(review_id, 1)
+      .then(() => {})
```

Comment:
If you don't want to do anything in the `.then` block you can remove it entirely. It'll still work without one 👍